subsystem/src/main/java/org/wildfly/extension/grpc/Priority.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright 2022 Red Hat
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wildfly.extension.grpc;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Used for specifying a priority for {@code ServerInteceptor}s
+ *
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface Priority {
```

Comment:
Any reason not to use `jakarta.annotation.Priority`?

Reply:
Actually, I was having trouble because the @Priority annotation was recognized, so I tried writing my own. Then I added jandex and the problem went away. I guess I should revert to `jakarta.annotation.Priority`.

Reply:
Fixed